src/components/InfoFooter.vue

```diff
@@ -31,22 +30,25 @@ export default {
   data() {
     return {
         environment: process.env.NODE_ENV,
-        release: "",
         repo: ""
       }
   },
   methods: {
+    getRelease() {
+      if (this.environment != 'development') {
+        return this.repo.slice(51, this.repo.length)
+      } else {
+        return this.repo.slice(38, this.repo.length)
+      }
+    },
     provider() {
       if (this.environment != 'development') {
         Promise.all([
```

Comment:
might be able to use the async/await commands https://dev.to/johnpaulada/synchronous-fetch-with-asyncawait  instead of Promise.all